quickstart/src/joinroom.js

```diff
@@ -47,7 +130,13 @@ function setActiveParticipant(participant) {
   // Attach the new active Participant's video.
   const { track } = Array.from(participant.videoTracks.values())[0] || {};
   if (track) {
-    track.attach($activeVideo.get(0));
+    const videoElement = $activeVideo.get(0)
+    if(track.mediaStreamTrack) {
+      window.CitrixWebRTC.mapVideoElement(videoElement);
+      videoElement.srcObject = window.CitrixWebRTC.createMediaStream([track.mediaStreamTrack]);
+    } else {
+      track.attach($activeVideo.get(0));
+    }
```

Comment:
Can we call mapVideoElement right after `attach` instead? That way customer won't have to manually create a mediastream. Will this work?

```ts
const el = track.attach($activeVideo.get(0));
window.CitrixWebRTC.mapVideoElement(el);
```

Reply:
Your suggestion would make it cleaner. Unfortunately, it does not work.